code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -138,12 +138,40 @@ const styles = {
   }
 };
 
-
 function dotClicked(url, e) {
   e.preventDefault();
   saveAnswersAndNavigate(url);
 }
 
+const BubbleInterior = React.createClass({
```

Comment:
I had actually recommended leaving it in the file. It's a pretty simple component that's not used anywhere else. (Could also be implemented as a stateless functional component).